matomo/rootfs/etc/confd/templates/setup-environment.sh.tmpl

```diff
@@ -17,11 +17,30 @@ foreground {
   MATOMO_DB_ROOT_PASSWORD="{{ getv "/db/root/password" "password" }}"
   MATOMO_DB_ROOT_USER="{{ getv "/db/root/user" "root" }}"
   MATOMO_DB_USER="{{ getv "/db/user" "matomo" }}"
-  MATOMO_SITE_HOST="{{ getv "/site/host" "islandora.isle-dc.localhost" }}"
-  MATOMO_SITE_NAME="{{ getv "/site/name" "Islandora" }}"
-  MATOMO_SITE_TIMEZONE="{{ getv "/site/timezone" "America/Halifax" }}"
+  MATOMO_DEFAULT_SITE_HOST="{{ getv "/default/site/host" "islandora.isle-dc.localhost" }}"
```

Comment:
This default should get changed to `islandora.traefik.me`, though i suppose that'll get covered in your other PR to set up the defaults.

Reply:
I'll do that in the other pull for sure, as I'll have to rebase it anyways after this is merged and make some changes.

Reply:
Hmm though might be easier to just set the default here then the isle-dc pull request can wait.